shared/src/commonMain/kotlin/com/bunbeauty/shared/data/repository/AdditionGroupRepository.kt

```diff
@@ -0,0 +1,16 @@
+package com.bunbeauty.shared.data.repository
+
+import com.bunbeauty.shared.data.mapper.additiongroup.mapAdditionGroupEntityToGroup
+import com.bunbeauty.shared.data.dao.addition_group.IAdditionGroupDao
+import com.bunbeauty.shared.domain.model.addition.AdditionGroup
+
+class AdditionGroupRepository(
+    private val additionGroupDao: IAdditionGroupDao,
+) {
+    suspend fun getAdditionGroup(uuid: String): AdditionGroup? {
+        return additionGroupDao.getAdditionGroupEntity(uuid)?.mapAdditionGroupEntityToGroup(
+            // don't need additions for current using
```

Comment:
But that's the only usage
Then we don't need it at all

Reply:
?